.gitignore

```diff
@@ -25,3 +25,4 @@ dist
 yarn.lock
 .env
 .idea
+__snapshots__/
```

Comment:
If we are going to make use of snapshot testing, then I believe we shall have to follow the rules in this area, one of which says these files should be added and committed to source control.

https://jestjs.io/docs/en/snapshot-testing#should-snapshot-files-be-committed

Given the not-very-straightforward nature of them though, I would suggest we avoid them until necessary.

Reply:
@Karuhanga, thanks for pointing this out. I stumbled upon it too but just like you have suggested, I figured we could save that for later when we are sure we won't have changes in the files causing us numerous obsolete snapshots.